src/centralserver/admin-ui/src/components/layout/AppToolbar.vue

```diff
@@ -33,12 +33,12 @@
     height="32"
     max-height="32"
   >
-    <div class="auth-container" v-if="isAuthenticated">
+    <div v-if="isAuthenticated" class="auth-container">
       <div class="server-type">X-ROAD CENTRAL SERVER</div>
```

Comment:
Security server was made mostly with plain flexbox etc. without Vuetity css classes. I think there are pros and cons with both ways. Like do we want to expand Vuetify dependency to everything, which way is easier for a newcomer to understand etc. There has not been an actual decision to go either way and the current situation is a bit confusing mix. I guess it would be good to discuss this more and maybe create some guidelines.  
For using Pug I would need a lot more convincing :D